test/irc/test_irc_isupport.py

```diff
@@ -162,10 +162,8 @@ def test_isupport_chanmodes():
 def test_isupport_chanmodes_undefined():
     instance = isupport.ISupport()
 
-    assert not hasattr(instance, 'CHANMODES')
-
-    with pytest.raises(AttributeError):
-        instance.CHANMODES
+    assert instance.CHANMODES.keys() == set("ABCD")
```

Comment:
Oops, I missed the py2 CI failure.

```suggestion
    assert set(instance.CHANMODES.keys()) == set("ABCD")
```

Reply:
\*counts the seconds to `python_requires = >=3.5`\*